protostar/starknet/cairo1_parser.py

```diff
@@ -0,0 +1,60 @@
+# pylint: disable="protected-access"
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any
+import json
+
+from starkware.cairo.lang.compiler.program import Program
+from starkware.starkware_utils.marshmallow_dataclass_fields import IntAsHex
+
+
+@dataclass(frozen=True)
+class CairoHintCode:
+    code: str
+    accessible_scopes: list[Any]
+    flow_tracking_data: Any
+
+
+@dataclass(frozen=True)
+class TestCase:
+    name: str
+    offset: int
+
+
+@dataclass(frozen=True)
+class TestSuite:
+    path: Path
+    test_cases: list[TestCase]
+    program: Program
+
+
+def parse_test_suite(path: Path, json_raw: Any) -> TestSuite:
```

Comment:
IMO, this function should return a data struct unrelated to testing concept.

Reply:
Why? The imput format is specifically designed around testing

Reply:
The name of this file is misleading then. It's not `cairo1_parser`, it's more `cairo1_test_suite_parser`.

Reply:
True that, I will change it